index.js

```diff
@@ -0,0 +1,76 @@
+$(".button-collapse").sideNav();
+
+var $tableBody = $("#tableBody");
+
+var cartedItem = {
+  itemName: "",
+  price: 0
+}
+
+bCartEmpty = true;
+
+var priceSubtotal = 0;
+var priceTotal = 0;
+var priceTax = 0;
+
+//event handler for image cards
+$cardGrid = $(".addToCart");
+$cardGrid.on("click", cbCardGrid);
+
+//event handler for form submit button
+var elmForm = document.getElementById("form");
+console.log(elmForm);
+elmForm.addEventListener('submit', function (event) {
```

Comment:
you seem to have attached a `submit` to a from element multiple times. Attaching multiple events to an element is fine, you should only attach one type of event to any element.